src/pages/my_registrations/MyRegistrations.tsx

```diff
@@ -44,6 +54,50 @@ export const MyRegistrations = ({ selectedUnpublished, selectedPublished }: MyRe
       return 0;
     });
 
+  const draftRegistrationMutation = useMutation({
```

Comment:
vurdere denne kanskje? `draftRegistrationMutation` kan like godt være å opprette en ny tenker jeg :p 
```suggestion
  const deleteDraftRegistrationsMutation = useMutation({
```